manager/integration/tests/common.py

```
@@ -4048,39 +4048,47 @@ def check_longhorn(core_api):
-            ready = True
+        except ApiException as e:
+            if (e.status == 404):
+                ready = False
 
-    except ApiException as e:
-        if (e.status == 404):
-            ready = False
+        time.sleep(RETRY_INTERVAL)
 
-    assert ready
+    assert ready, f"Failed to wait for Longhorn components ready"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)